Catalog.html

```diff
@@ -19,6 +20,127 @@
+            <ul>
+              <li>
+                <input type="checkbox" name="pool" id="pool" checked>
+            <label for="pool">Бассейн</label>
+              </li>
+              <li>
+                <input type="checkbox" name="parking" id="parking">
+            <label for="parking">Парковка</label>
+              </li>
+              <li>
+                <input type="checkbox" name="WI-FI" id="WI-FI" checked>
+            <label for="WI-FI">WI-FI</label>
+              </li>
+            </ul>
+          </fieldset>
+          <fieldset>
+            <legend>Тип жилья</legend>
+            <ul>
+              <li>
+                <input type="checkbox" name="hotel" id="hotel" checked>
+              <label for="hotel">Гостиница</label>
+              </li>
+              <li>
+                <input type="checkbox" name="motel" id="motel" checked>
+              <label for="motel">Мотель</label>
+              </li>
+              <li>
+                <input type="checkbox" name="apartments" id="apartments" checked>
+              <label for="apartments">Апартаменты</label>
+              </li>
+            </ul>
+          </fieldset>
+          <fieldset>
+            <legend>Стоимость в сутки, &#8381;</legend>
+            <div class="filter">
+              <div class="filter-interval">
+                <label>От <input type="text" name="min-interval" value="0"></label>
+                <label>До <input type="text" name="max-interval" value="3000"></label>
+              </div>
+              <div class="filter-range-wrapper">
+                <div class="filter-range">
+                  <div class="filter-range-scale">
+                    <div class="filter-range-bar"></div>
+                  </div>
+                  <button class="filter-range-handle min"></button>
+                  <button class="filter-range-handle max"></button>
+                </div>
+              </div>
+            </div>
+          </fieldset>
+          <button type="submit">Показать</button>
+          </form>
+          </section>
+          <section>
+            <h2>Найдено:</h2>
+            <p>3</p>
+            <h3>Сортировка</h3>
+            <ul>
+              <li>
+                <a href="">По цене</a>
+              </li>
+              <li>
+                <a href="">По типу</a>
+              </li>
+              <li>
+                <a href="">По рейтингу</a>
+              </li>
+            </ul>
+            <ul>
+              <li>
+                <a href=""></a>
+              </li>
+              <li>
+                <a href=""></a>
+              </li>
+            </ul>
+          </section>
+          <section>
+            <h2>Список отелей</h2>
+            <ul>
+              <li>
+                <h3>Amara resort & spa</h3>
+                <p>Гостиница от 4000</p>
+                <img src="image\hotel with red trees.jpg" width="135" height="90" alt="фото отеля с красными деревьями">
```

Comment:
```suggestion
                <img src="image/hotel_with_red_trees.jpg" width="135" height="90" alt="фото отеля с красными деревьями">
```
В названии файла не должно быть пробелов.